cpp/src/binaryop/binaryop.cpp

```diff
@@ -285,9 +286,22 @@ void binary_operation(gdf_column* out, gdf_column* lhs, gdf_column* rhs,
         reinterpret_cast<NVCategory*>(temp_rhs.dtype_info.category));
     return;
   }
+
+  gdf_column lhs_tmp;
+  gdf_column rhs_tmp;
```

Comment:
Always initialize, even if default. 
```suggestion
  gdf_column rhs_tmp{};
```

Reply:
:+1: